PassleSync.Core/PropertyEditors/PassleTagsPropertyEditor.cs

```diff
@@ -0,0 +1,17 @@
+﻿using PassleSync.Core.Constants;
+using Umbraco.Core.Logging;
+using Umbraco.Core.PropertyEditors;
+using Umbraco.Web.PropertyEditors;
+
+namespace PassleSync.Core.PropertyEditors
+{
+    [TagsPropertyEditorAttribute]

```

Comment:
Is there a reason this attribute is needed for this property editor but the other property editors don't require attributes other than the `DataEditorAttribute`?